__tests__/app.test.js

```diff
@@ -167,6 +167,32 @@ describe('/api/articles', () => {
             expect(message).toBe('Invalid Query')
         })
     })
+    test('GET 200: Should return the array sorted by the provided query in the provided order', () => {
+        return request(app)
+        .get('/api/articles?sort_by=votes&order=asc')
+        .expect(200)
+        .then(({ body: { articles }}) => {
+            expect(articles).toBeSortedBy('votes', { 
+                ascending: true
+            })
+        })
+    })
+    test('GET 404: Should return an appropriate status and error message if provided an invalid sort_by', () => {
```

Comment:
I would argue that this would be a 400 for a bad request rather than something is missing.